From c144aaca88e04124a28e10a485738cddff99c819 Mon Sep 17 00:00:00 2001
From: mbrinkma <mbrinkma@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 10 Jan 2019 16:29:35 +0000
Subject: [PATCH 345/462] MANTIS 0041015 BuildManagement: use new Gradle 5.1
 because the former (3.4.1) now uses testng-7.0.0beta3 which has errors

git-svn-id: svn://localhost/SelfServiceBuild/trunk@614 2eb42a87-3827-0410-b767-dc15055982e4
---
 .../Build/scripts/gradle/wrapper/gradle-wrapper.properties      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/scripts/gradle/wrapper/gradle-wrapper.properties b/SelfServiceBuild/Build/scripts/gradle/wrapper/gradle-wrapper.properties
index 8d1f6c6..ac1377c 100644
--- a/SelfServiceBuild/Build/scripts/gradle/wrapper/gradle-wrapper.properties
+++ b/SelfServiceBuild/Build/scripts/gradle/wrapper/gradle-wrapper.properties
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1-bin.zip
-- 
2.41.0.windows.1

